src/PureCloud.Client/Apis/LearningApi.cs

```diff
@@ -29,20 +54,240 @@ public async Task<LearningAssignment> GetLearningAssignmentAsync(string assignme
 
         if (expand?.Any() == true)
         {
-            parameters.Add("expand", string.Join(",", expand));
+            parameters.Add("expand", string.Join(",", expand.Select(UriHelper.ParameterToString)));
+        }
```

Comment:
Add multiple values instead of gluing them. Also just do null check without calling Any()